client/data/partition_service_region.json

```diff
@@ -4553,6 +4553,31 @@
             "us-west-2": {}
           }
         },
+        "ses": {
```

Comment:
Was this block added manually or from the region generator? (If manually, it will be overridden next time the generator runs)

Reply:
Manually. Could you point me to the generator?

Reply:
replaced with proper `email` entry